### PR TITLE
Upgrade to version 1.8.12

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *New version upstream*

## Solution
- *Upgrade to version 1.8.12*
- *Remove email send to admin #118*
- *New permissions*
- *Remove plugin versioning*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/etherpad_mypads_ynh%20PR116%20(ericgaspar)/)  

Failing on change_url script...
```
Package linter:                  SUCCESS
Installation:                    SUCCESS
Deleting:                        SUCCESS
Installation in a sub path:      SUCCESS
Deleting from a sub path:        SUCCESS
Installation on the root:        SUCCESS
Deleting from root:              SUCCESS
Upgrade:                         SUCCESS
Installation in private mode:    SUCCESS
Installation in public mode:     SUCCESS
Multi-instance installations:    Not evaluated.
Malformed path:                  Not evaluated.
Port already used:               SUCCESS
Backup:                          SUCCESS
Restore:                         SUCCESS
Change URL:                      FAIL
Actions and config-panel:        FAIL
```

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -226,7 +226,7 @@ chown -R $app: $final_path
 (cd "$final_path/src"
 ynh_exec_warn_less npm cache clean --force
 # Remove package-lock.json to prevent any shit during npm update...
-ynh_secure_remove --file="$final_path/src/package-lock.json"
+#ynh_secure_remove --file="$final_path/src/package-lock.json"
```
Comment (alexAubin) [inline scripts/upgrade:229]: You sure about that considering the comment just before ? (No idea what exactly that may refer to)

**Review by alexAubin — APPROVED** [review]
Super quick review, LGTM

File: scripts/upgrade (hunk 1 above)
Comment (ericgaspar) [reply scripts/upgrade]: OK, trying again with this line uncommented, but this was previously failing upgrade tests...

**Comment by ericgaspar** [comment]
!testme

**Comment by squattertim** [comment]
So, what's next? = )

**Comment by alexAubin** [comment]
Not sure why so many 502 over the place ... I restarted the job in case that was false negatives ...

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
I don't know either ...
I find this package complicated to work because when you touch something everything explodes (the installation procedure and the systemd settings seem to have evolved in the new versions of Etherpad). There is also the version plugin system which adds complexity and is not necessarily relevant; the installation of plugins and their updates is well managed in the Etherpad interface.
I have been working on a new version of etherpad-lite (without plugin and with postgresql as database) it available here: https://github.com/YunoHost-Apps/etherpad_ynh I do not know if it is relevant to add it to the catalog or if it will just create confusion.

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by maniackcrudelis** [comment]
Been ping by the YunoHost-Apps/apps-group...

Just an advice, consider package-lock.json removing and the versioning of modules.
Things are hardly done more complicated than they should be just for fun.

This app is far more tricky than it looks like.

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**